examples/energy-management-app/energy-management-common/energy-evse/include/EnergyEvseDelegateImpl.h

```diff
@@ -56,33 +56,47 @@ enum EVSEStateMachineEvent
 class EvseSession
 {
 public:
-    EvseSession(EndpointId aEndpoint) { mEndpointId = aEndpoint; }
+    EvseSession() {}
     /**
      * @brief This function records the start time and provided energy meter values as part of the new session.
      *
+     * @param aEndpointId           - The endpoint to report the update on
      * @param chargingMeterValue    - The current value of the energy meter (charging) in mWh
      * @param dischargingMeterValue - The current value of the energy meter (discharging) in mWh
      */
-    void StartSession(int64_t chargingMeterValue, int64_t dischargingMeterValue);
+    void StartSession(EndpointId aEndpointId, int64_t chargingMeterValue, int64_t dischargingMeterValue);
```

Comment:
`aEndpointId` seems to follow a pattern of `aArgumentName` that other source files use. However in this file we don't seem to use that, so a name of `endpointId` seems more appropriate.

This applies throughout.

Reply:
ok changed